cmd/utils/flags.go

```diff
@@ -1705,6 +1717,11 @@ func SetEthConfig(ctx *cli.Context, stack *node.Node, cfg *ethconfig.Config) {
 		}
 		cfg.Genesis = core.DefaultScrollAlphaGenesisBlock()
 		// SetDNSDiscoveryDefaults(cfg, params.ScrollAlphaGenesisHash)
+	case ctx.GlobalBool(ScrollSepoliaFlag.Name):
+		if !ctx.GlobalIsSet(NetworkIdFlag.Name) {
+			cfg.NetworkId = 534351
```

Comment:
Can we explicitly set `l1.confirmations` and `l1.sync.startblock` here?

Reply:
sure, will do.

Reply:
done -- https://github.com/scroll-tech/go-ethereum/pull/377/commits/fe1bbba6c477ca1d1b1336e4b81507b72e11d6b2